Wikipedia/Code/WMFAppViewController+Extensions.swift

```diff
@@ -1012,11 +1074,12 @@ extension WMFAppViewController {
     }
     
     @objc func updateActivityTabLoginState(activityTabViewController: WMFActivityTabViewController) {
+        // todo grey
```

Comment:
Just wanted to highlight this todo in case you didn't intend for it to stay here.